domain/macaroon/state/types.go

```diff
@@ -5,11 +5,38 @@ package state
 
 import (
 	"database/sql/driver"
+	"time"
 
 	"github.com/go-macaroon-bakery/macaroon-bakery/v3/bakery"
+	"github.com/go-macaroon-bakery/macaroon-bakery/v3/bakery/dbrootkeystore"
 	"github.com/juju/errors"
 )
 
+type RootKey struct {
```

Comment:
This should not be exported. Also add a comment.

Reply:
If I want to remove dbrootkeystore from the state layer, this will have to be exported so the service layer can use it.

I may move this into a domain-level `types.go` file

Reply:
Done

Reply:
Had to re-export as a result of other changes. But only between service and state